index.js

```diff
@@ -741,6 +743,45 @@ function createPanZoom(domElement, options) {
       return zoomByRatio(clientX, clientY, scaleMultiplier)
   }
 
+
+  /**
+   * Calculate the center of a given bounding rectangle's position from our container viewpoint
+   * @param {DOMRect} elemBounds 
+   */
+  function getCenterOfBounds(elemBounds) {
+    const containerBounds = owner.getBoundingClientRect()
+
+    const centerX = -elemBounds.left + (((containerBounds.width / 2) - (elemBounds.width / 2)))
+    const centerY = -elemBounds.top + (((containerBounds.height / 2) - (elemBounds.height / 2)) + containerBounds.top)
+
+    const newX = transform.x + centerX
+    const newY = transform.y + centerY
+
+    return { x: newX, y: newY }
+  }
+
+  /**
+   * Moves the view to the center of element
+   * @param {Element} element get the center of this HTML element
+   * @param {Number} xOffset offset x pixels from center horizontally
+   * @param {Number} yOffset offset y pixels from center vertically
+   */
+  function moveToCenterOfElement(element, xOffset = 0, yOffset = 0) {
+    const bounds = element.getBoundingClientRect()
```

Comment:
is `element` part of the nested tree? 

Note: panzoom works not only with dom elements, but also with canvas/svg/webgl - having element passed directly like this would likely make the API less generic. Ideal solution would delegate this logic into [domController](https://github.com/anvaka/panzoom/blob/master/lib/domController.js) or [svgController](https://github.com/anvaka/panzoom/blob/master/lib/svgController.js)

Also note: panzoom uses older version of ES language. Having consts, destructuring, default values would break the minifier